static/js/flow/searchTypes.js

```diff
@@ -0,0 +1,6 @@
+// @flow
+import type { Profile } from './profileTypes';
+
+export type ProfileResult = {
```

Comment:
would it make more sense to call this `SearchResult` or something? we are eventually going to have our own API endpoint that returns us some serialized search result info, and i imagine it will have more information than just the Profile